content/collections.md

```diff
@@ -0,0 +1,375 @@
+---
+title: Collections and Models
+---
+
+After reading this guide, you'll know:
+
+1. What the different flavors of Mongo Collection in Meteor are, and how to use them.
```

Comment:
That becomes awkward when you try and write "MongoDB database" though. Could probably work around that though...